shared/src/packages/business/libs/types/service/business-create-payload.type.ts

```diff
@@ -0,0 +1,10 @@
+import { type UserEntity as UserEntityT } from '~/packages/users/libs/types/types.js';
+
+import { type BusinessEntity as BusinessEntityT } from '../request/request.js';
+
+type BusinessCreatePayload = {
+  payload: Omit<BusinessEntityT, 'id' | 'ownerId'>;
+  owner: Pick<UserEntityT, 'id'> & { group: { key: string } };
```

Comment:
```suggestion
  owner: Pick<UserEntityT, 'id'> & { group: { key: ValueOf<typeof UserGroupKey >;} };
```
but actually it's better to create separate type for it